src/main/java/com/github/fridujo/rabbitmq/mock/MockChannel.java

```diff
@@ -58,6 +60,17 @@ public MockChannel(int channelNumber, MockNode node, MockConnection mockConnecti
         this.node = node;
         this.mockConnection = mockConnection;
         this.metricsCollectorWrapper = metricsCollectorWrapper;
+
+        node
```

Comment:
Not sure why this queue has to be declared, given it is substituted in consume / get methods

Reply:
some high level apis like spring RabbitMqTemplate use queueDeclarePassive to ensure the queue exists, to work around this i declare it in advance

Reply:
In this case, why not making just an `if` statement in the `queueDeclarePassive` method, like you do in basicGet / basicPublish ?

To be honest I am not confortable creating a **Queue** for this as the documentation states it is a "pseudo-queue".
I am afraid someone debugging the whole thing, find it to be a queue, but not behaving like one.
It is quite confusing.

Reply:
I thought creating the queue would be bulletproof but this is just because i lack experience in amqp to foresee possible consequences. Anyway i agree with your concerns and changed the code to plain if statement. Please take a look